packages/webdriverio/tests/utils/interception/devtools.test.ts

```diff
@@ -24,6 +24,7 @@ const cdpClient: any = {
 const browserMock = {} as any as Browser
 
 const fetchListener = async (mock: any, params: any, client = cdpClient) => {
+    const mocks = Array.isArray(mock) ? mock : [mock]
```

Comment:
So we could test interception logic with multiple mocks

Reply:
I am not entirely sure anymore if this has been implemented for easier testing or to test a feature.

Reply:
This function is used in almost all the tests in this file (except for the cases we need to test request and response behaviour separately), so I guess it's just for easier testing.